wp-content/plugins/core/src/Blocks/Blocks_Subscriber.php

```diff
@@ -83,6 +84,13 @@ public function register(): void {
 			return $filter ? $filter->filter_block_content( $block_content, $block ) : $block_content;
 		}, 10, 2 );
 
+		/**
+		 * Add a default css class name to specific blocks.
+		 */
+		add_filter( 'render_block', function ( string $block_content, array $parsed_block, object $block ): string {
+			return $this->container->get( Add_Block_Default_Class_Name::class )->add_class_name( $block_content, $parsed_block, $block );
+		}, 10, 3 );
```

Comment:
@ChrisMKindred , would love it if you can check this filter and make sure that it's not duplicating some way we could handled this with the filter factory methods a bit further up in this subscriber.